opentechinstitute/commotion-feed
# Dashboard helper rewrite

To be tested in conjunction with opentechinstitute/commotion-dashboard-helper#10

Updates dependencies for commotion dashboard helper.
To test:
1. During build, after running setup but before running make, ensure that openwrt/packages/cyassl DOES NOT EXIST.
2. After build, ensure that the cyassl (libcyassl?) package in openwrt/bin/ar71xx/packages/ shows version 2.8, not 1.6.5
3. After flashing node, ensure that the node's https pages display and function correctly (since this is a major openssl update)
4. Continue with dashboard helper test procedures.
